github-for-jira.sd.yml

```diff
@@ -4,6 +4,8 @@ description: Connect app for integrating GitHub into Jira
 organization: Engineering-DevOps
 notifications:
   email: fusion-arc@atlassian.com
+tags:
+  brahmos: enabled
```

Comment:
/nit
Can you please help me understand, that this tag will not impact the performance on our prod system? Or unless we tell brahmos to run on prod, everything will be as usuall?

Reply:
brahmos isn't supported in prod yet so if we could tag it for only stage that would be ideal! 
we have some `environmentOverrides` in there already, but never tried to access that top level values :shrug:

Reply:
@joshkay10 I think that we cannot apply global props on environmentOverrides

Reply:
@joshkay10 some updates. It works!

Reply:
Hi @yevhshevchenko so this is back to global settings now?

Reply:
Hi @gxueatlassian, yeah it was a problem when I tried to execute it and faced with an error(